index.js

```diff
@@ -0,0 +1,54 @@
+// Main Validate Password Function
+const validatePassword = password => {
+  let isMinLength = checkMinLength(password)
+  let isLowerCase = false
+  let isUpperCase = false
+  let isNumeric = false
+  let isSpecial = false
+
+  for (let i = 0; i < password.length; i++) {
+    !isLowerCase ? isLowerCase = checkLowerCase(password[i]) : isLowerCase = true
```

Comment:
not the cleanest way to go about it... 

1. can use if statements
2. isLowerCase = !isLowerCase ? checkLowerCase(password[i]) : true
3. isLowerCase = isLowerCase || checkLowerCase(isLowerCase) ||  false